InformationScripting/src/precompiled.h

```diff
@@ -44,6 +44,7 @@
 
 // Put here includes which appear in header files. This will also be visible to other plug-in which depend on this one
 // and will be included in their precompiled headers
+#include <QtCore/QCommandLineParser>
```

Comment:
Does this have to appear in a header file? If not, move it to the lower section of `precompiled.h`